db/institutions.yml

```diff
@@ -403,7 +403,7 @@ caltech:
   state: CA
   zip: '91125'
 umontreal:
-  aka: [UdeM, udem]
+  aka: [UdeM, udem, umontreal, UMontreal, UofMontreal, uofmontreal]
```

Comment:
you don't need umontreal because we also search on id and that is the id.  Please add the ones that I put in the other comment too, ie., the name written out in full in various ways, but not in french.  THanks

Reply:
the way to understand the aka is it is all the ways that this might be written, so we need to capture "University of Montreal" at the very least

Reply:
Thaks! Now I have a better idea about this field.